File: api/Medico.Api/DB/Models/AppUser.cs
using System;
using System.Collections.Generic;

namespace Medico.Api.DB.Models
{
    public class AppUser : BaseEntity
    {
        public string Hash { get; set; }

        public string Login { get; set; }

        public bool IsSuperAdmin { get; set; }

        public Guid? EmployeeId { get; set; }

        public Employee Employee { get; set; }

        public List<AppUserPermissionGroup> UserPermissionGroups { get; set; }
    }
}

File: client/src/dataModels/appUser.ts
import { BaseEntityModel } from "./baseEntityModel";

export class AppUser extends BaseEntityModel {
    Hash: string;
    Login: string;
    IsSuperAdmin: boolean;
    EmployeeId: string;

    constructor(id: string = "", isDelete: boolean = false) {
        super(id, isDelete);

        this.Hash = "";
        this.Login = "";
        this.IsSuperAdmin = false;
        this.EmployeeId = "";
    }

    createFromEntityModel(entity: any) {
        const appUser = new AppUser(entity.Id, !!entity.IsDelete);

        appUser.Hash = entity.Hash;
        appUser.Login = entity.Login;
        appUser.IsSuperAdmin = entity.IsSuperAdmin;
        appUser.EmployeeId = entity.EmployeeId;

        return appUser;
    }
}